components/common/shoppingCart/ShoppingCartItemRow.vue

```diff
@@ -99,4 +114,10 @@ onMounted(() => {
 .inherit-background-color {
   background-color: inherit !important;
 }
+.gap-8 {
```

Comment:
It's unnecessary to set `gap`. 'mb-2' could help you do that

Reply:
The context I am using the gap is for adding space between flex siblings that are side by side.
I would not favor margins because margins would add space whether the elements are visible or not.
While gap only adds the space when both are visible.

This implementation is based on the fact that item's visibilities are constantly toggled.
Also, the directions are horizontal and not vertical.

Let me know what you think.

Reply:
Just avoiding some unnecessary custom style. Adding `ml-2` to the right element could do the same.
<img width="1040" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/99d2dba9-b292-4b30-9bfb-f9387436a6bd">